cmake/onednn.cmake

```diff
@@ -18,9 +18,9 @@ if (NOT DEFINED GC_DNNL_INCLUDES)
     # This allows to generate headers from *.in without adding the library to the build.
     # If the build is required, remove this and the SKIP_ADD option above.
     if (DEFINED CMAKE_GENERATOR)
-        set(GENERATOR_FLAG "-G ${CMAKE_GENERATOR}")
+        set(GENERATOR_FLAG "${CMAKE_GENERATOR}")
     endif ()
-    execute_process(COMMAND ${CMAKE_COMMAND} ${GENERATOR_FLAG}
+    execute_process(COMMAND ${CMAKE_COMMAND} -G ${GENERATOR_FLAG}
```

Comment:
what if `CMAKE_GENERATOR` is not defined? It seems that we'll get here an invalid command anyway: 
```
cmake -G _empty_ -Wno-dev ...
```

Reply:
Fixed by adding an extra var for `-G`.